pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonDefaults.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.gradle.python.extension;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+import org.gradle.api.GradleException;
+
+
+public class PythonDefaults {
```

Comment:
I don't think this is PythonDefault's, since it only contains version information about python versions you can use. Maybe `PythonDefaultVersions`?